pact/Pact/Core/IR/Term.hs

```diff
@@ -542,26 +544,7 @@ traverseDefTerm f = \case
   DPact d -> DPact <$> traverseDefPactTerm f d
 
 
-instance Plated (Term name ty builtin info) where
-  plate f = \case
-    Var n i -> pure (Var n i)
-    Lam ns term i -> Lam ns <$> f term <*> pure i
-    Let n t1 t2 i -> Let n <$> f t1 <*> f t2 <*> pure i
-    App t1 t2 i -> App <$> f t1 <*> traverse f t2 <*> pure i
-    Builtin b i -> pure (Builtin b i)
-    Constant l i -> pure (Constant l i)
-    Sequence e1 e2 i -> Sequence <$> f e1 <*> f e2 <*> pure i
-    Conditional o i ->
-      Conditional <$> traverse f o <*> pure i
-    ListLit m i -> ListLit <$> traverse f m <*> pure i
-    Nullary term i ->
-      Nullary <$> f term <*> pure i
-    CapabilityForm cf i ->
-      CapabilityForm <$> traverse f cf <*> pure i
-    Try e1 e2 i ->
-      Try <$> f e1 <*> f e2 <*> pure i
-    ObjectLit o i ->
-      ObjectLit <$> (traverse._2) f o <*> pure i
+instance (Data ty, Data name, Data builtin, Data info) => Plated (Term name ty builtin info)
```

Comment:
Could've also done
```
deriving (..., Plated)
```
at `Term` definition site, but `DeriveAnyClass` isn't generally used in the codebase, so dunno if you'd be happy with that.